tests/test_yaml.py

```diff
@@ -0,0 +1,93 @@
+import importlib
+import os
+from unittest import mock
+import pytest
+
+from rpmlb.yaml import Yaml
+import helper
+
+
+@pytest.fixture
+def valid_yaml():
+    """A Yaml object with a valid YAML file."""
+    return Yaml(helper.get_valid_recipe_file())
+
+
+def test_init_loads_file():
+    yaml = Yaml(helper.get_valid_recipe_file())
+    assert yaml
+
+
+def test_init_load_default_loader_when_failing_to_import_c_loader():
```

Comment:
What I want to ask you is how to write a unit test for `rpmlb/yaml.py` the line: `from yaml import Loader` in the `__init__` function.

if you see `htmlcov/rpmlb_yaml_py.html` with your browser, you can see the line is not covered by the unit test.